### DISCUSSION TITLE
CSS Styling for Grouping

### DISCUSSION THREAD

**Discussion by gitgrube** [punxaphil/custom-sonos-card#521] [Q&A] [answered]
Hello, 

I just want to set my Grouping card as shown in Example 1. I copied the following [https://github.com/punxaphil/custom-sonos-card?tab=readme-ov-file#grouping](url)
For me, however, it only changes as follows:
![image](https://github.com/user-attachments/assets/0e1d309a-8a89-4ecc-9b8c-a47dc75c006e)

What am I doing wrong? Unfortunately I am not an expert in CSS Styling.

Best regards, 
Lorenz

**Comment by punxaphil** [comment, marked as answer]
I'm considering removing that example from the documentation. It was created by JaseOnline, and I am guessing he uses a totally different theme for HA.
You could try asking him in the HA forums. Here is the link to his post https://community.home-assistant.io/t/dashboard-sonos-card/393620/658?u=punxsutawneyphil
